src/test/java/com/artipie/front/settings/YamlCredentialsTest.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * The MIT License (MIT) Copyright (c) 2022 artipie.com
+ * https://github.com/artipie/front/LICENSE.txt
+ */
+package com.artipie.front.settings;
+
+import com.amihaiemil.eoyaml.Yaml;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Test case for {@link YamlCredentials}.
+ * @since 1.0
+ */
+@SuppressWarnings("PMD.AvoidDuplicateLiterals")
+final class YamlCredentialsTest {
+
+    @Test
+    void findUser() {
+        final var username = "John";
+        final var user = new YamlCredentials(
+            Yaml.createYamlMappingBuilder()
+                .add(
+                    "credentials",
+                    Yaml.createYamlMappingBuilder()
+                        .add(
+                            username,
+                            Yaml.createYamlMappingBuilder().build()
+                        ).build()
+                ).build()
```

Comment:
this code is repeating in all three tests, I'd suggest to extract method to build this yaml, the method can accept userename and yaml mapping as parameters